roles/handle-constraints-url/tasks/main.yaml

```diff
@@ -23,4 +23,4 @@
       TOX_CONSTRAINTS_FILE: "{{ constraints_file.path }}"
       # Backward compatibility, to be removed
       UPPER_CONSTRAINTS_FILE: "{{ constraints_file.path }}"
-  when: constraints_stat.size > 0
+  when: constraints_file is defined and constraints_stat.size > 0
```

Comment:
it should be `constraints_stat.stat.size`, it's still missing the `.stat`

Reply:
+1

Reply:
I just pushed a new revision of the change to address this.